tests/daemon/test-machine-id-provider.c

```diff
@@ -115,33 +115,6 @@ teardown (MachineIdTestFixture *fixture,
 
 // Testing Cases
 
-static void
-test_machine_id_provider_create_tracking_id_if_unavailable (MachineIdTestFixture *fixture,
-                                                            gconstpointer         tdata)
-{
-  g_autoptr(GError) error = NULL;
-  g_autofree gchar *contents = NULL;
-
-  g_assert_false (g_file_test (fixture->tracking_id_file_path, G_FILE_TEST_EXISTS));
-
-  g_autoptr(EmerMachineIdProvider) id_provider =
-    emer_machine_id_provider_new_full (fixture->tracking_id_file_path);
-
-  uuid_t id;
-  // id_provider_get_id will write a new tracking ID, if no ID is found.
-  g_assert (emer_machine_id_provider_get_id (id_provider, NULL, id));
-
-  g_assert (g_file_test (fixture->tracking_id_file_path, G_FILE_TEST_EXISTS));
```

Comment:
I missed this last time but perhaps we should keep this test case, but adjust it to assert that get_id fails if the file doesn't exist, and g_file_test (..., EXISTS) is still false after the call. This will exercise the "file doesn't exist" path.

Reply:
Let's check failing to get the tracking-id.